org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/Messagebox.java

```diff
@@ -387,15 +399,19 @@ else if (event.getTarget() == btnIgnore)
 		{
 			returnValue = IGNORE;
 		}
-
+		close();
+	}
+	
+	private void close() {
+		SessionManager.getSessionApplication().getKeylistener().removeEventListener(Events.ON_CTRL_KEY, this);
```

Comment:
Just a question - why is this line required?  Is zk adding the listener internally when you add the annotation?

Reply:
Oops. I first tried a different approach using a the KeyListener. That worked equally well but was a bit more complicated so I removed it. Except for this line which I just had forgotten. - will be removed